elf/module.go

```diff
@@ -770,3 +784,14 @@ func (b *Module) CloseExt(options map[string]CloseOptions) error {
 	}
 	return nil
 }
+
+// UpdateKprobeNameForHandler takes a handler and try to find the corresponding kprobe from a previously loaded mapping,
+// if found then update the kprobe name to the value of newKprobeName
+func (b *Module) UpdateKprobeNameForHandler(handler, newKprobeName string) error {
```

Comment:
we should make it explicit in the function name and documentation that `handler` is a section name.

Reply:
`UpdateKprobeNameForSecHandler`? It cannot be any longer 🤣

Reply:
How about `SetKprobeForSection` ?

Reply:
OK that could work. Although given that they mix all terms together, nothing seems to be able remove the confusion completely.